src/components/molecules/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,139 @@
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { useState, FC, FormEvent, ChangeEvent } from 'react';
+
+import Button, { EButtonTheme } from '@components/atoms/Button';
+import Input from '@components/atoms/Input';
+import { ETypographyVariant } from '@enums/typography';
+import { useAuth } from '@src/contexts/AuthContext';
+import { Auth } from '@types';
+
+import * as S from './LoginForm.styled';
+
+interface LoginProps {
+  type: string;
+}
+
+const LoginForm: FC<LoginProps> = ({ type }) => {
```

Comment:
Slight rename as the form handles both `Login` and `Sign Up`
```suggestion
const AuthForm: FC<LoginProps> = ({ type }) => {
```